src/Components/SavedSearchAlert/Utils/extractPills.ts

```diff
@@ -180,34 +180,42 @@ export const extractPillsFromCriteria = ({
         break
       }
       case "includeArtworksByFollowedArtists": {
-        result = {
-          field: paramName,
-          value: paramValue,
-          displayValue: "Artists You Follow",
+        if (paramValue) {
```

Comment:
I wonder if we could extract this check above, before the `switch` statement, to avoid duplicating code